.github/workflows/test.yml

```diff
@@ -15,8 +15,8 @@ jobs:
       fail-fast: false
       max-parallel: 5
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10']
-        django-version: ['2.2', '3.0', '3.1', '3.2', '4.0', 'main']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        django-version: ['2.2', '3.0', '3.1', '3.2', '4.0', '4.1', '4.2', 'main']
```

Comment:
Can you remove `3.0`, `3.1` and `4.1` from here? Otherwise this matrix is getting a bit too bit.

Reply:
I assume you mean `4.0` instead of `4.1` (which I've just added)?

Reply:
Yes, that, thanks!